python/tk_multi_publish2/context_widget/context_widget.py

```diff
@@ -695,7 +695,7 @@ def _query_related_tasks(self, context):
             [["entity", "is", context.entity]],
             # query all fields required to create a context from a task entity
             # dictionary. see sgtk api `context_from_entity_dictionary`
-            fields=["type", "id", "name", "project", "entity", "step"]
+            fields=["type", "id", "name", "content", "project", "entity", "step"]
```

Comment:
Maybe this list of field could be turned into a constant so both methods always use the updated list?